### PR TITLE
fix(electron): compatible with windows

### LINKED ISSUE #146: Cannot open standalone app
在已经全局安装 `@vue/devtools` 的情况下，运行 `vue-devtools` 没有弹出 devtool 窗口

+ `node v21.4.0`
+ `npm v10.2.5`
+ `@vue/devtools v7.0.4`

ps: 尝试运行了一下 `--debug`，竟然有输出，以供参考

```shell
PS > vue-devtools --debug
file:///E:/Code/Environment/Node/node_global/node_modules/@vue/devtools/node_modules/execa/lib/error.js:60
                error = new Error(message);
                        ^

Error: Command failed with exit code 9: E:\Code\Environment\Node\node_global\node_modules\@vue\devtools\node_modules\electron\dist\electron.exe /E:/Code/Environment/Node/node_global/node_modules/@vue/devtools/node_modules/@vue/devtools-electron/dist/app.cjs --debug
    at makeError (file:///E:/Code/Environment/Node/node_global/node_modules/@vue/devtools/node_modules/execa/lib/error.js:60:11)
    at execaSync (file:///E:/Code/Environment/Node/node_global/node_modules/@vue/devtools/node_modules/execa/index.js:201:17)
    at file:///E:/Code/Environment/Node/node_global/node_modules/@vue/devtools/node_modules/@vue/devtools-electron/dist/cli.mjs:6:16
    at ModuleJob.run (node:internal/modules/esm/module_job:218:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:329:24) {
  shortMessage: 'Command failed with exit code 9: E:\\Code\\Environment\\Node\\node_global\\node_modules\\@vue\\devtools\\node_modules\\electron\\dist\\electron.exe /E:/Code/Environment/Node/node_global/node_modules/@vue/devtools/node_modules/@vue/devtools-electron/dist/app.cjs --debug',
  command: 'E:\\Code\\Environment\\Node\\node_global\\node_modules\\@vue\\devtools\\node_modules\\electron\\dist\\electron.exe /E:/Code/Environment/Node/node_global/node_modules/@vue/devtools/node_modules/@vue/devtools-electron/dist/app.cjs --debug',
  escapedCommand: '"E:\\Code\\Environment\\Node\\node_global\\node_modules\\@vue\\devtools\\node_modules\\electron\\dist\\electron.exe" "/E:/Code/Environment/Node/node_global/node_modules/@vue/devtools/node_modules/@vue/devtools-electron/dist/app.cjs" --debug',
  exitCode: 9,
  signal: undefined,
  signalDescription: undefined,
  stdout: undefined,
  stderr: undefined,
  cwd: 'C:\\Users\\19278',
  failed: true,
  timedOut: false,
  isCanceled: false,
  killed: false
}

Node.js v21.4.0
```

### PR DESCRIPTION
**Pull request by alexzhang1030**
fixes #146

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -63,6 +63,9 @@
     "zip": "tsx ./scripts/extension-zip.ts",
     "gen:vue-apis": "tsx ./scripts/vue-api-manifest.ts"
   },
+  "dependencies": {
+    "pathe": "^1.1.1"
```
Comment (webfansplz) [inline package.json]: We should move the dep to the electron package.

Comment (alexzhang1030) [reply package.json]: Yes, I just did lol

**Comment by alexzhang1030** [comment]
I think we can merge now

**Review by webfansplz — APPROVED** [review]

**Comment by webfansplz** [comment]
Great work!

**Pull request merged**